tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "module": "esnext",
     "moduleResolution": "node",
     "resolveJsonModule": true,
-    "isolatedModules": false,
+    "isolatedModules": true,
```

Comment:
Nice - Just curious, do you think will this need to be changed back down the line?